File: Laborator/Lab 3/JavaClientServer/Networking/src/main/java/dto/AdminDto.java
package dto;

import java.io.Serializable;

public class AdminDto implements Serializable {

    public String getId() {
        return id;
    }

    public String getPass() {
        return pass;
    }

    private String id;
    private String pass;
    private String name;

    public AdminDto(String id){
        this(id,"","");
    }
    public AdminDto(String id,String pass){
        this(id,pass,"");
    }
    public AdminDto(String id, String pass,String name) {
        this.id = id;
        this.pass = pass;
        this.name=name;
    }

    public String getName() {
        return name;
    }
}

File: Laborator/Lab 3/C# Client-server/Networking/Dto/AdminDto.cs
using System;

namespace Networking.Dto
{
	[Serializable]
	public class AdminDto
	{
		private string id;
		private string password;

		public AdminDto(string id) 
            : this(id, "") {}

		public AdminDto(string id, string password)
		{
			this.id = id;
			this.password = password;
		}

		public virtual string ID
		{
			get { return id; }
			set { id = value; }
		}

		public virtual string Password
		{
			get { return password; }
		}
	}

}